dbt_transformation_modify.go

```diff
@@ -0,0 +1,138 @@
+package fivetran
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"time"
+)
+
+// DbtTransformationModifyService implements the dbt transformation management,
+// modify a dbt atransformation api
+// Ref. PATCH https://api.fivetran.com/v1/dbt/transformations/{transformation_id}
+
+type DbtTransformationModifyService struct {
+	c                   *Client
+	dbtTransformationId *string
+	schedule            *DbtTransformationSchedule
+	runTests            *bool
+}
+
+type dbtTransformationModifyRequestBase struct {
+	RunTests *bool `json:"run_tests"`
+}
+
+type dbtTransformationModifyRequest struct {
+	Schedule *dbtTransformationScheduleRequest `json:"schedule,omitempty"`
+	dbtTransformationModifyRequestBase
+}
+
+type DbtTransformationModifyResponseDataBase struct {
+	ID              string    `json:"id"`
+	DbtModelId      string    `json:"dbt_model_id`
+	OutputModelName string    `json:"output_model_name`
+	DbtProjectId    string    `json:"dbt_project_id"`
+	LastRun         time.Time `json:"last_run"`
+	NextRun         time.Time `json:"next_run"`
+	Status          string    `json:"status"`
+	RunTests        *bool     `json:"run_tests"`
+	ConnectorIds    []string  `json:"connector_ids"`
+	ModelIds        []string  `json:"model_ids"`
+}
```

Comment:
Same here, we could use common struct for all responses

Reply:
done. :)